### PR TITLE
 #3292 Create pipeline graphs (connector,stream)

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3292 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
@wu87988622 would you mind fixing the GIt conflicts and commit again 🙏

File: ohara-manager/client/src/components/Pipeline/Graph/Connector/styles.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+import { css } from 'styled-components';
+
+export const connector = ({ theme }) => css`
+  background-color: white;
+  position: absolute;
+  border-style: solid;
+  border-radius: ${theme.shape.borderRadius}px;
+  border-color: ${theme.palette.divider};
+  border-width: 1px;
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Graph/Connector/styles.js]: `border: 1px solid ${theme.palette.divider}`

File: ohara-manager/client/src/components/Pipeline/Graph/Connector/ConnectorGraph.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      const modelId = this.model.id;
+
+      this.$box
+        .find('button#remove')
+        .on('click', _.bind(this.model.remove, this.model));
+
+      this.$box.find('button#link').on('mousedown', function() {
+        linkLine = new joint.dia.Link();
+        linkLine.source({ id: modelId });
+        linkLine.attr({
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Graph/Connector/ConnectorGraph.js]: @wu87988622 Please leave some comment here for better explaining the intent of the code :)

File: ohara-manager/client/src/components/Pipeline/Graph/Connector/ConnectorGraph.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+      '</div>',
+    ].join(''),
+    initialize() {
+      _.bindAll(this, 'updateBox');
+      joint.dia.ElementView.prototype.initialize.apply(this, arguments);
+
+      this.$box = $(_.template(this.template)());
+
+      // Update the box position whenever the underlying model changes.
+      this.model.on('change', this.updateBox, this);
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Graph/Connector/ConnectorGraph.js]: This line is necessary, right?

File: ohara-manager/client/src/components/Pipeline/Graph/Connector/styles.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/Pipeline/Graph/Connector/styles.js]: ok thanks

File: ohara-manager/client/src/components/Pipeline/Graph/Connector/ConnectorGraph.js (hunk 3 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/Pipeline/Graph/Connector/ConnectorGraph.js]: yes

File: ohara-manager/client/src/components/Pipeline/Graph/Connector/ConnectorGraph.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/Pipeline/Graph/Connector/ConnectorGraph.js]: ok

**Pull request merged**